# response of a robot, if it wants to take a certain task and if so what its score is

string request_id
string robot_id
bool reject
float32 score
